native/android-jsi/src/main/java/com/nozbe/watermelondb/jsi/WatermelonJSI.java

```diff
@@ -0,0 +1,14 @@
+package com.nozbe.watermelondb.jsi;
+
+import android.app.Application;
+
+// Public interface to JSI-based Watermelon
+public class WatermelonJSI {
+    public static void install(Application context, long javaScriptContextHolder) {
+        JSIInstaller.install(context, javaScriptContextHolder);
+    }
+
+    public static void onTrimMemory(int level) {
+      // TODO: Unimplemented
+    }
```

Comment:
What do you want to do in `onTrimMemory` later?

Reply:
well… trim… memory :) remove sqlite caches, stuff like that.